README.md

```diff
@@ -64,13 +64,16 @@ For a listing of development commands use the following:
     npm run                   # list npm commands 
     gulp help                 # list gulp commands
 
-### Deploying
+### Releasing
 
-    npm version <version>     # bump the new version
-    git push                  # push bump commit
-    git push --tags           # push version tag
+On the latest clean `master`:
+
+    npm run release:major
+    npm run release:minor
+    npm run release:patch
```

Comment:
Releases are now easy and explicit.  Thank to our ta-scripts package, they are also consistent across our projects.